model/project_ref.go

```diff
@@ -2434,6 +2434,44 @@ func GetMessageForPatch(patchID string) (string, error) {
 	return project.CommitQueue.Message, nil
 }
 
+func ValidateContainers(project string, containers []Container) error {
+	pRef, err := FindMergedProjectRef(project, "", true)
+	if err != nil {
+		return errors.Errorf("Unable to find merged project ref for project %s", project)
```

Comment:
There's a new error style (which you can probably see in the rest of the files if you rebase/merge), so errors involving DB lookups should be consistent with the new style.

```go
return errors.Wrapf(err, "finding merged project ref '%s'", project)
```

This comment might not matter at all depending on what's done with [this comment](https://github.com/evergreen-ci/evergreen/pull/5558/files#r846445422), but the new error style is the way to go.